turtlebot3_fake_node/param/waffle_pi.yaml

```diff
@@ -0,0 +1,10 @@
+turtlebot3_fake_node:
+  ros__parameters:
+    wheels:
+      separation: 0.287
+      radius: 0.033
+
+    joint_states_frame: "base_footprint"
+    odom_frame: "odom"
+    base_frame: "base_footprint"
+    
```

Comment:
There is no EOF.

Reply:
now() function is used in the very next line to update the variable 'prev_update_time_'. I hope that is what he meant? @routiful